sway-core/src/asm_generation/fuel/programs/abstract.rs

```diff
@@ -195,12 +209,18 @@ impl AbstractProgram {
                     comment: "data section offset".into(),
                     owning_span: None,
                 },
+                // word 3 -- full word u64 placeholder
+                AllocatedAbstractOp {
+                    opcode: Either::Right(ControlFlowOp::ConfigurablesOffsetPlaceholder),
+                    comment: "configurables offset".into(),
+                    owning_span: None,
+                },
                 AllocatedAbstractOp {
                     opcode: Either::Right(ControlFlowOp::Label(label)),
-                    comment: "end of metadata".into(),
+                    comment: "end of configurables offset".into(),
```

Comment:
It looks like this used to signal the end of the data section. Maybe this should still be `end of metadata` since now it signifies the end of both?

Reply:
We may want to add more metadata later, let's not write something that may be invalidated later.

Reply:
This is just a comment indicating the end of where the current offset section ends. It doesn't invalidate anything we may do later.